src/components/app.jsx

```diff
@@ -1,9 +1,48 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import CreateCards from './create-cards'
+import ReviewCards from './review-cards'
+import ViewCards from './view-cards'
+import Nav from './nav'
 
 class App extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            view: 'view-cards',
+            cssBadge: ''
+        }
+        this.setView = this.setView.bind(this);
+        this.getView = this.getView.bind(this);
+    }
+
+    setView(newView){
+        this.setState({
+            view: newView,
+            cssBadge: newView
+        })
+        console.log(this.state.cssBadge)
```

Comment:
Remember that a `console.log` right after `setState` won't be accurate. `setState` is asynchronous. Also you should remove all `console.log`'s before making a pull request.